airflow/jobs/scheduler_job.py

```diff
@@ -453,21 +453,19 @@ def manage_slas(self, dag: DAG, session: Session = None) -> None:
             notification_sent = False
             if dag.sla_miss_callback:
                 # Execute the alert callback
-                self.log.info(' --------------> ABOUT TO CALL SLA MISS CALL BACK ')
+                self.log.info('Calling SLA miss callback')
                 try:
                     dag.sla_miss_callback(dag, task_list, blocking_task_list, slas,
                                           blocking_tis)
                     notification_sent = True
                 except Exception:  # pylint: disable=broad-except
-                    self.log.exception("Could not call sla_miss_callback for DAG %s",
-                                       dag.dag_id)
-            email_content = """\
+                    self.log.exception("Could not call sla_miss_callback for DAG %s", dag.dag_id)
+            email_content = f"""\
             Here's a list of tasks that missed their SLAs:
             <pre><code>{task_list}\n<code></pre>
             Blocking tasks:
-            <pre><code>{blocking_task_list}\n{bug}<code></pre>
-            """.format(task_list=task_list, blocking_task_list=blocking_task_list,
-                       bug=asciiart.bug)
+            <pre><code>{blocking_task_list}<code></pre>
+            """
```

Comment:
I'm wondering if we should remove the indentation for this string:
```
In [1]: x = """\
   ...:             Here's a list of tasks that missed their SLAs:
   ...:             <pre><code>{task_list}\n<code></pre>
   ...:             Blocking tasks:
   ...:             <pre><code>{blocking_task_list}\n{bug}<code></pre>
   ...:             """

In [2]:

In [2]: print(x)
            Here's a list of tasks that missed their SLAs:
            <pre><code>{task_list}
<code></pre>
            Blocking tasks:
            <pre><code>{blocking_task_list}
{bug}<code></pre>
```

Reply:
https://docs.python.org/3/library/textwrap.html?highlight=dedent#textwrap.dedent maybe?

Reply:
Oh, though this is HTML, so whitespace doesn't really matter.

Reply:
@ashb yeah that's HTML so indeed it's not a problem as long as the email client can render it